types/json.go

```diff
@@ -104,22 +107,28 @@ func (a JSONArray) String() string {
 		}
 		switch {
 		case a.IsNull(i):
-			o.WriteString("(null)")
+			o.WriteString(array.NullValueStr)
 		default:
-			fmt.Fprintf(o, "\"%s\"", arr.Value(i))
+			fmt.Fprintf(o, "\"%s\"", a.ValueStr(i))
```

Comment:
I do think that this is wrong (as JSON can have quotes in it, so we should use `%q` here instead)

Reply:
Addressed in 8ea6a1c

Reply:
Yeah, doesn't look right to me either